src/lib.rs

```diff
@@ -31,55 +31,73 @@
 //!
 //! ## Finding image files in the current directory.
 //!
-//! ```no_run
+//! ```rust
 //! extern crate globwalk;
+//! # include!("doctests.rs");
```

Comment:
Huh, that is a neat idea for extra test code!  I feel like the this should be a tip documented somewhere.

Reply:
[Oh but it is documented as pro tip!](https://killercup.github.io/rustfest-idiomatic-libs/index.html#/4/3) ;)

Reply:
Glossed over that last time I looked at your presentation.  Ideally we find permanent central homes for these things so people don't have to dig through all of the past presentations / blog posts and hope its still the latest.